projects/so-manual.halex.alignment.xml

```diff
@@ -1312,7 +1312,7 @@ _target{&#13;
                 <child name="title" ns="http://www.w3.org/1999/xlink"/>
             </property>
         </target>
-        <parameter value="AX_Bodenschätzung" name="value"/>
+        <parameter value="AX_Bodenschaetzung" name="value"/>
```

Comment:
Doesn't this need to be lowercase?

Reply:
Sorry, that is wrong. It is the value for the target property and no feature type name in this case.